src/server/server_family.cc

```diff
@@ -1533,6 +1535,11 @@ void ServerFamily::ReplConf(CmdArgList args, ConnectionContext* cntx) {
         string sync_id = absl::StrCat("SYNC", sid);
         cntx->conn_state.repl_session_id = sid;
 
+        if (!cntx->replica_conn) {
+          ServerState::tl_connection_stats()->num_replicas += 1;
+        }
+        cntx->replica_conn = true;
+
```

Comment:
You do this inside PSYNC for Redis.

Does a replica count if its still setting up flows and preparing to do full sync (which should be a very short phase)? I guess, yes